### PR TITLE
TSPS-280 Update Teaspoons e2e test to work with GCP workspaces

### PR DESCRIPTION
**Pull request by mmorgantaylor**
Goes along with this Teaspoons PR: https://github.com/DataBiosphere/terra-scientific-pipelines-service/pull/127

Successful run: https://github.com/DataBiosphere/terra-scientific-pipelines-service/actions/runs/10928088257/job/30336118144

Ticket: https://broadworkbench.atlassian.net/browse/TSPS-280

### CODE REVIEW

File: e2e-test/tsps_azure_e2e_test.py [hunk 1]
```diff
@@ -200,12 +200,12 @@ def download_with_azcopy(sas_url):
...
 found_exception = False
 try:
-    logging.info("Starting Workflows Azure E2E test...")
+    logging.info("Starting Teaspoons Azure E2E test...")
     logging.info(f"billing project: {billing_project_name}, env_string: {env_string}")
 
     # Create workspace
     logging.info("Creating workspace...")
-    workspace_id, workspace_name = create_workspace(billing_project_name, azure_user_token, rawls_url, workspace_name)
+    workspace_id, workspace_name = create_workspace(billing_project_name, azure_tsps_sa_token, rawls_url, workspace_name)
```
Comment (mmorgantaylor) [inline e2e-test/tsps_azure_e2e_test.py:208]: this should be the admin token

File: e2e-test/tsps_azure_e2e_test.py [hunk 2]
```diff
@@ -216,11 +216,6 @@ def download_with_azcopy(sas_url):
     logging.info(f"adding tsps qa service account to billing project {billing_project_name}")
     add_user_to_billing_profile(rawls_url, billing_project_name, tsps_sa_email, azure_user_token)
```
Comment (mmorgantaylor) [inline e2e-test/tsps_azure_e2e_test.py:217]: this should also be the admin token, not the user token

File: e2e-test/tsps_gcp_e2e_test.py [hunk 3]
```diff
@@ -147,45 +144,91 @@ def upload_file_with_azcopy(sas_url):
...
+    response = requests.post(uri, headers=headers)
+    
+    if response.status_code != 201:
+        raise Exception(response.text)
+    
+    return response.json()['groupEmail']
+
+
+def add_member_to_terra_group(orch_url, group_name, email_address, role, token):
+    formatted_email = email_address.replace("@", "%40")
```
Comment (jsotobroad) [inline e2e-test/tsps_gcp_e2e_test.py:198]: can probably use urlencode here in case there are other weird characters in the values

**Review by jsotobroad — APPROVED** [review]
one small comment

**Pull request merged**